sktime/transformations/series/boxcox.py

```diff
@@ -211,9 +211,18 @@ def _inverse_transform(self, X, y=None):
 class LogTransformer(BaseTransformer):
     """Natural logarithm transformation.
 
-    The natural log transformation can used to make data more normally
+    The Natural logarithm transformation can be used to make the data more normally
     distributed and stabilize its variance.
 
+    Transforms each data point x to (scale * log(x+offset))
```

Comment:
this does not agree with what the function does. I suggest changing the docstring to what the function does.